packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -363,23 +187,11 @@ ScrollableBase.defaultProps = {
  * @ui
  * @public
  */
-const Scrollable = Skinnable(
-	SpotlightContainerDecorator(
-		{
-			overflow: true,
-			preserveId: true,
-			restrict: 'self-first'
-		},
-		I18nContextDecorator(
-			{rtlProp: 'rtl'},
-			ScrollableBase
-		)
-	)
-);
+const Scrollable = {};
```

Comment:
Leave some note why `Scrollable` became a null object.

Reply:
I updated it.